test/callingResWriteHeadDirectly.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const hijackResponse = require("../lib/hijackResponse");
+const { Readable } = require("stream")
+const expect = require("unexpected")
+  .clone()
+  .use(require("unexpected-express"))
+  .addAssertion(
+    "<object> to yield response <object|string>",
+    (expect, subject, value) =>
+      expect(subject, "to yield exchange", {
+        request: "GET /",
+        response: value
+      })
+  );
+
+describe("calling res.writeHead", function() {
+  it.only("simple case with status code", function() {
```

Comment:
```suggestion
  it("simple case with status code", function() {
```

😏

Reply:
hehe... yeah, this was not supposed to land like this. just needed it online quickly to have a reference to share. I don't plan on merging in this state ;)

Also, we need to move to GitHub Actions. :)